data/colors_v2/themes/dark_high_contrast.ts

```diff
@@ -4,7 +4,7 @@ import dark from './dark'
 const scale = {
   black: '#010409',
   white: '#ffffff',
-  gray: ['#ffffff', '#f1f3f6', '#d7dbdf', '#b9bfc6', '#98a1ad', '#79828e', '#5b616b', '#272b33', '#272b33', '#090c11'],
+  gray: ['#ffffff', '#f0f3f6', '#d9dee3', '#bdc4cc', '#9ea7b3', '#7a828e', '#525964', '#272b33', '#272b33', '#00a0c10'],
```

Comment:
Looks like the build failed because `#00a0c10` is not a valid color

Reply:
Broken color is now fixed to `#0a0c10`